files-community/Files
# Feature: Added `ctrl + alt + c` shortcut to copy path with quotes

**Resolved / Related Issues**
Closes: #12545 

**Validation**
How did you test these changes?
   1. Select item and use `ctrl + alt + c` shortcut
   2. Confirm the path is copied to the clipboard with quotes

**Screenshots (optional)**
Add screenshots here.
